docs/root/intro/arch_overview/http/http_connection_management.rst

```diff
@@ -252,6 +256,11 @@ and then shipped upstream with all of the normal Envoy request sanitization taki
   second route, even if they are the same, so be careful configuring header modification rules to
   avoid duplicating undesired header values.
 
+.. warning::
+  Note that no filters will see the response that triggers the internal redirect. If there is a need
```

Comment:
No _downstream_ filters will.  You actually can see and modify headers with upstream filters

Reply:
Right.  I'll update with that.